### ISSUE TITLE
[feature]  capability to set Host and port , and even basePath 

### ISSUE THREAD

**Issue by LasneF** [OWASP/OFFAT#70]
Hello 

given than an OAS spec can be used in various implementation ( dev , uat, prod) 
the server section is not always accurate 

it would be great to be able to override the setting with a command like this 

offat -f .\openapi.yml http://myHost:myPort/myPath 

notice it is not as same as setting a proxy

this use case it taken into consideration for tooling such as prism or vaccum

**Comment by dmdhrumilmistry** [comment]
> Hello
> 
> given than an OAS spec can be used in various implementation ( dev , uat, prod) the server section is not always accurate
> 
> it would be great to be able to override the setting with a command like this
> 
> offat -f .\openapi.yml http://myHost:myPort/myPath
> 
> notice it is not as same as setting a proxy
> 
> this use case it taken into consideration for tooling such as prism or vaccum

Hi @LasneF,

would below option work for you?

```
offat -f openapi.yml --server http://host:port/path
```

if host flag is not provided then it'll expect value to be present in the OAS file.

Let me know if I'm interpreting this incorrectly along with few reference links.

**Comment by LasneF** [comment]
this would make the deal yes , then is it by default , is it --host , or server , you are the boss :)

**Comment by dmdhrumilmistry** [comment]
> this would make the deal yes , then is it by default , is it --host , or server , you are the boss :)

It'll be `server` flag

**Comment by dmdhrumilmistry** [comment]
this feature has been released in latest version of offat `0.17.0`.

you can bump to latest version using

```bash
python -m pip install -U offat
```
> replace `python` with `python3` if required
